modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2TeamMembersRemoveEditRoles.testcase

```diff
@@ -0,0 +1,321 @@
+			CP2Utils.openActionMenu(
+				menuItem = "Remove",
+				openUser = "Test User");
+
+			Button.click(button = "Remove");
+		}
+
+		task ("Assert the success message when submitting the form") {
+			Alert.viewSuccessMessage();
+		}
+
+		task ("Delete the entries and objects created") {
+			CustomerPortal.deleteEntryAccountFlag(entry = "ERC-002");
+		}
+
+		task ("Delete user created") {
+			JSONUser.deleteUserByEmailAddress(userEmailAddress = "test.user@liferay.com");
+		}
+	}
+
+	@description = "Automation ID: LPS-150012 | Test Summary: Verify that after the user clicks Edit a Save button is available"
+	@priority
+	test CanViewSaveButton {
+		task ("Change the user role in the project") {
+			Accounts.openAccountsAdmin();
+
+			CustomerPortal.accountTableProject(title = "Test Account DXP");
+
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
+
+		task ("Go to CP2.0 and login to okta") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+
+			CustomerPortal.loginOktaToUpdateCP();
+
+			SelectWindow(locator1 = "title=Home - Customer Portal Site - Liferay DXP");
+
+			Refresh();
+		}
+
+		task ("Get into the project and go to Team Members page") {
+			CP2TeamMembers.teamMembersFlow();
+		}
+
+		task ("Open the action menu and click the Edit option") {
+			CP2Utils.openActionMenu(
+				menuItem = "Edit",
+				openUser = "Test Test");
+		}
+
+		task ("Change the guest user role and save") {
+			Button.click(button = "Save");
+		}
+
+		task ("Delete the entries and objects created") {
+			CustomerPortal.deleteEntryAccountFlag(entry = "ERC-002");
+		}
+	}
+
+	@description = "Automation ID: LPS-150007 | Test Summary: Verify if by clicking Edit the user can change a function already selected among the available functions in the drop-down"
+	@priority = "4"
+	test ChangeAnAlreadySelectedRole {
+		task ("Change the user role in the project") {
+			Accounts.openAccountsAdmin();
+
+			CustomerPortal.accountTableProject(title = "Test Account DXP");
+
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
+
+		task ("Go to CP2.0 and login to okta") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+
+			CustomerPortal.loginOktaToUpdateCP();
+
+			SelectWindow(locator1 = "title=Home - Customer Portal Site - Liferay DXP");
+
+			Refresh();
+		}
+
+		task ("Get into the project and go to Team Members page") {
+			CP2TeamMembers.teamMembersFlow();
+		}
+
+		task ("Open the action menu and click the Edit option") {
+			CP2Utils.openActionMenu(
+				menuItem = "Edit",
+				openUser = "Test Test");
+		}
+
+		task ("Click on the drop-down and make sure the role are enabled") {
+			CP2TeamMembers.assertDropDownOptionsEnabled();
+		}
+
+		task ("Delete the entries and objects created") {
+			CustomerPortal.deleteEntryAccountFlag(entry = "ERC-002");
+		}
+	}
+
```

Comment:
Do you think this can be combined with another test?

Reply:
With the changes you asked for above, I believe I can put this CanViewSaveButton test together